frontend/js/behaviors/core/dropdown.js

```diff
@@ -197,6 +197,10 @@ const dropdown = function(container) {
   function _init() {
     document.addEventListener('mediaQueryUpdated', _mediaQueryUpdated, true);
     _mediaQueryUpdated();
+
+    window.addEventListener('popstate', function(event) {
```

Comment:
Interesting, I'd never seen this event before.

Reply:
@zachgarwood https://developer.mozilla.org/en-US/docs/Web/API/Window/popstate_event